### ISSUE TITLE
Демо счет для демонстрации возможностей приложения

### ISSUE THREAD

**Issue by vananiev** [spacious-team/investbook#615]
Завести пример отчета, загрузив который можно посмотреть возможности приложения.

**Comment by EPoY74** [comment]
ПриветстВую!
Задагние актуально?

Сколько данные необходимо? Есит реальные данные, но их мало (пока).

**Comment by vananiev** [comment]
Да, сейчас особенно важно для разработки UI. В демосчете нужно нагенерить данные, это собо важно.
